### DISCUSSION TITLE
Can I do something to help?

### DISCUSSION THREAD

**Discussion by LyubomirT** [caffeine-addictt/waku#118] [Ideas]
Hey there @caffeine-addictt! I've finally received a break at work and am able to work on some side projects and help people. Is there something I can do to improve the repository?

**Comment by caffeine-addictt** [comment]
Hey @LyubomirT, it's good to hear from you! Hope you've been well.

## Developments

Perhaps I'll start with a small recap since #28.

I ported the Javascript code to Typescript in #34 and made QoL changes until #80. (see [release](https://github.com/caffeine-addictt/waku/releases/tag/node-v1.x.x)). Also outlined in #75 is my decision to move from a node-based script to a distributed binary with **Golang** and the general idea I have on how this software will function. I also have rebranded this project, previously called **template**, to **waku** in #105 and #80.

## Current WIP

There is still fair bit of things to do, but I'll split them into Golang and non-Golang work.

### Golang

- ~~Post-template setup scripts ([docs](https://github.com/caffeine-addictt/waku/blob/main/docs/templating.md#setup))~~ removed as too big of a security issue for me to maintain for
- Platform specific label creation ([docs](https://github.com/caffeine-addictt/waku/blob/main/docs/templating.md#labels))
- ~~General testing (app-level)~~
- ~~Code structure improvements (#146)~~
- ~~Fixing bugs for license templating (#126)~~

### Non-Golang

- ~~Documentation on Waku commands~~
- ~~Documentation on how to create good styles~~
- Official template styles we provide (within `./template` of this repo) - Probably the most important
- ~Making waku more downloadable on windows (our chocolatey release is still being reviewed, considering moving to a scoop bucket)~ - I created a [scoop bucket](https://github.com/caffeine-addictt/scoop-bucket)


---

That should be it for the more important stuff for now. No pressure or anything of course haha, even just using waku for your own projects and reporting bugs is more than enough ❤️

**Reply by LyubomirT** [reply]
Got it, I'll get to one of these when I'm done studying, thank you for your detailed response!

**Reply by caffeine-addictt** [reply]
> Official template styles we provide (within `./template` of this repo) - Probably the most important

@LyubomirT
Hey just an update. I'd say this is what Waku needs most right now.
I've also wrote the documentation for templating and creating styles (except waku commands), so it'll be easier on you (i hope?)

On other news, I'd say Waku is already pretty usable for your project templates too.
